### DISCUSSION TITLE
Larger block text (pseudo large terminal banner/heading font)

### DISCUSSION THREAD

**Discussion by ardgraft** [Textualize/rich#1018] [Ideas]
I would like support for block types of different sizes. To allow big titles, banners etc. 
I hacked [this](https://www.dropbox.com/s/qldmdepsbkkgcr2/ardgraft-homelab.png.jpg?dl=0) together in python to support my home labs.  **Rich** would benefit with something like that.

**Comment by willmcgugan** [comment]
That would be a nice feature. It probably doesn't belong in the core library, just because I need to draw a line somewhere. But it could be an external lib. If you would like to tackle it yourself, its fairly straightforward to add an [integration](https://rich.readthedocs.io/en/latest/protocol.html).
